packetbeat/docs/fields.asciidoc

```diff
@@ -9996,7 +9996,7 @@ type: array
 --
 Chain of trust for the client certificate.
 
-type: array
```

Comment:
I also think this one is an array of objects so it will need to remain as an `array` (essentially dynamically mapped). And there cannot be any aliases pointing directly at this field as per https://www.elastic.co/guide/en/elasticsearch/reference/current/alias.html#alias-targets.